### PR TITLE
move `.md` files to published package

### PR DESCRIPTION
**Pull request by SergeAstapov**
This moves `README.md`, `LICENSE.md` and `CHANGELOG.md` from the repository root into the `./addon` directory.

In doing so, the three files will be picked up by `npm pack` and `npm publish` respectively, so that the npmjs.com listing shows the proper `README.md` instead of an error message, and so that these files are also available in the users `node_modules`.

It also adds symlinks from the old location to the new location, so that GitHub still renders the repository overview correctly. It's important, that the files are physically moved instead of just symlinked into the package, as `npm pack` doesn't follow symlinks.

`CONTRIBUTING.md` is intentionally not moved, as it's only relevant in the context of the repository.

### CODE REVIEW

**Comment by SergeAstapov** [comment]
related to https://github.com/embroider-build/embroider/pull/1188

**Comment by knownasilya** [comment]
TIL about markdown symlinking on GH.

**Comment by SergeAstapov** [comment]
@knownasilya do you think this can be merged and published as v8.0.0? At this point of time there is only v8.0.0-beta.0 with v2 addon format conversion and wonder if anything left for the actual release to happen

File: test-app/package.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
     "ember-page-title": "8.0.0-beta.0",
     "ember-qunit": "^5.1.5",
     "ember-resolver": "^8.0.3",
-    "ember-source": "~3.28.3",
+    "ember-source": "~3.20.5",
```
Comment (bertdeblock) [inline test-app/package.json:57]: Was this change intended?

Comment (SergeAstapov) [reply test-app/package.json:57]: indeed no! I'm not sure how it happened 🤦
